.gitignore

```diff
@@ -1,7 +1,5 @@
-Debug
-Release
-MinSizeRel
```

Comment:
Why remove these?

Reply:
I didn't see how anybody could still be using them; am I wrong?

Reply:
I build in subdirectories named Debug and Release, though I see I have them in my global .gitignore so I guess it doesn't affect me.

Reply:
I'll restore them; thanks.